### PR TITLE
Add additional rounding for labels of Y-axis when it's needed

### PR DESCRIPTION
**Pull request by Andrey321**

### CODE REVIEW

**Comment by racurry** [comment]
I don't know that elroi should make assumptions about how to skew the data.  If the values coming in are too small for a precision of 0, it seems like it should be the caller's responsibility to pass in an appropriate precision.

**Comment by sshymov** [comment]
LGTM

**Comment by dkrishu** [comment]
:shipit:

File: elroi.js [hunk 1]
```diff
@@ -1304,21 +1304,34 @@
...
-         * @param {String} unit The units of the data
+         * This draws either the y1 or y2 axis, depending on the series data.
+         * @param {object} The axis for witch labels should be drawn.
          */
-        function drawYLabels(maxVal, minVal, axis) {
+        function drawYLabels(axis) {
 
             // Draw the y labels
             var $yLabels = $('<ul></ul>')
                 .addClass("y-ticks")
```
Comment (taqi) [inline elroi.js]: use ' or " but everywhere.

**Comment by gsmith85** [comment]
I see 5 test failures in our test suite.  Pop open test/index.html and you should see them.

File: elroi.js [hunk 2]
```diff
@@ -1304,21 +1304,34 @@
...
+
+            // The graph can contain only small values and scale ratio can be less than one
+            // (more than one pixel is used per a value unit). And in combination with zero precision this can leads
+            // to situation when labels with fractional numbers are too over-aggressively rounded and it looks confusing.
+            // So in that case precision will be increased.
+            if (precision === 0
+                // scale ratio check
+                && graph.yTicks[axis.seriesIndex] > 1
+                // labels contain fractional numbers check
+                && ((maxVal + Math.abs(minVal)).toFixed(0) % (graph.options.grid.numYLabels - 1)) !== 0) {
```
Comment (gsmith85) [inline elroi.js]: Why not jut use `maxVal - minVal`, that will correctly calculate the range for the entire set of real numbers?

Comment (Andrey321) [inline elroi.js]: It makes sense. thank you.

**Pull request merged**